.env.example

```diff
@@ -50,6 +50,8 @@ SALESFORCE_SECRET_KEY=topsecret
 MAILER_BASE_URI=https://mailer-staging.thebiggivetest.org.uk
 MAILER_SEND_SECRET=
 
+ACCOUNT_MANAGEMENT_BASE_URI=http://localhost:4200
```

Comment:
Will need corresponding var in infrastructure

Reply:
I'm not sure it does - the base URI only changes rarely, I think the way we've managed these links in Donate or Wordpress seems to have worked well, i.e. having one hardcoded for each of three envs, local, staging, regression, and prod. 

How about adding a function to get this URI on the Environment enum https://github.com/thebiggive/matchbot/blob/4c2fff3f2bc830ac10c6a0afcc6f1482f395b59b/src/Application/Environment.php ?

Reply:
If you do think we need the env var could we make it something like `DONATE_BASE_URI` instead of `ACCOUNT_MANAGEMENT_BASE_URI` , so we can re-use it in future if we want to generate links to e.g. campaigns.

Reply:
Thinking again there is some value in having it in env vars as it would be shared with `donate_default_uri_prefix` used by identity. Maybe worth setting up vars for `experienceUriPrefix` and `blogUriPrefix` at the same time and passing them to all our main services, e.g. so we can use them for presenting links directly in mailer in future.

Reply:
If you make an change in infrastructure that will take us to code in **7** repos modified for one story.